sinks/grpsink/state_helper_test19.go

```diff
@@ -0,0 +1,63 @@
+// +build go1.9
+
+package grpsink
+
+import (
+	"context"
+	"sync/atomic"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"google.golang.org/grpc/connectivity"
+)
+
+func waitThroughStateSequence(t *testing.T, sink *GRPCStreamingSpanSink, dur time.Duration, seq ...connectivity.State) {
+	t.Helper()
+
+	first, current := seq[0], sink.grpcConn.GetState()
+	if !assert.ObjectsAreEqual(first, current) {
```

Comment:
this smells super strongly like it wants to be `require.Equal` (:

Reply:
comment added - it doesn't use `require.Equal` because then the use of `t.Helper()` above would be pointless, because those libraries haven't yet been updated to use `t.Helper()`.